docs/backplane/requirements/acs/10-acs-managed-service.md

```diff
@@ -32,5 +43,38 @@ Permissions at cluster scope.
 * get pods and pod logs
 * view deployments
 * get deployments
+* view routes
+* view egress firewalls
+* view roles and rolebindings
+* patch persistentvolumeclaims
+
+ACS team needs to inspect the `cloud-service-sensible-declarative-configs` secret
```

Comment:
What is in that secret and why does the team need to have access to read it?  Is this something required for day-to-day support?

Reply:
This is for customer demands to enable multiple organizations access to an ACSCS instance as per https://gitlab.cee.redhat.com/stackrox/acs-cloud-service/runbooks/-/blob/master/sops/dp-034-allow-multiple-orgs-access.md?ref_type=heads

Reply:
There isn't anything in that SOP about access to this secret.  I assume this is sensitive data.  What is the reason to have any read permisisons on the secret?  If it's to verify something has landed in cluster is it normal to do that with normal day-to-day?  Is there another way (some downstream effect) to determine if the changes have landed?

Reply:
You're right. A [commit](https://gitlab.cee.redhat.com/stackrox/acs-cloud-service/runbooks/-/commit/b9d334fe929b429ab9567d26cb2be73e7e927a63) was pushed 1 hours ago 🥹 with a different SOP. Will remove